### DISCUSSION TITLE
cool! - autoreceive?

### DISCUSSION THREAD

**Discussion by tezoatlipoca** [SpatiumPortae/portal#88] [Ideas]
This is very cool. Is there anyway I wonder to leverage portal to perform an auto-receive? Like setup the receiver as a daemon to receive files? Fix the magic password to something only known to the sender. 

I.e. I have a webserver at home, I don't want to expose ssh through my firewall, I'd like to expose a relay server at <obscure non-obvious> port and just shoot files home whenever I wanted by using `portal send -relay:home.IP:obscport predefinedmagicpwd <files>`?

**Comment by ZinoKader** [comment]
Thank you!

The part that I can get behind is to be able to set your own password. The autoreceiving and all of the rest could easily be achieved with some shell scripting to fit your use case.

Will open an issue about custom passwords.
